### ISSUE TITLE
Unable to access custom fields via GQL

### ISSUE THREAD

**Issue by jpcarpenter** [verbb/hyper#71]
### Describe the bug

Perhaps I'm misunderstanding something, but I have been unable to access custom fields using GraphQL. 

Here is the custom field I've added:
<img width="623" alt="Screen Shot 2023-06-22 at 10 35 30 PM" src="https://github.com/verbb/hyper/assets/5934150/94fac2d3-1b86-4ea1-9a9f-293c7706fe64">

Here is the test I've conducted within the GraphQL explorer within Craft:
<img width="1659" alt="Screen Shot 2023-06-22 at 10 39 54 PM" src="https://github.com/verbb/hyper/assets/5934150/e990f2b4-3f10-4e90-85dd-eea92e11a6af">

I've also tried the solutions pointed out in #69 thinking it was possibly related, but had no success.

### Steps to reproduce

1. Add a custom field to a Hyper fields Social Media link type field settings.
2. Attempt to access field via GraphQL.

### Craft CMS version

4.4.14

### Plugin version

1.1.7

### Multi-site?

No

**Comment by engram-design** [comment]
Refer to https://github.com/verbb/hyper/discussions/68 but you will need to use latest (unreleased) changes to have access to the link type as a fragment.

```gql
{
  entries(id: 123) {
    ... on blog_blog_Entry {
      
      hyperFieldHandle {
        linkUrl

        ... on hyperFieldHandle_Url_LinkType {
          plainText
        }
      }
    }
  }
}
```

**Comment by jpcarpenter** [comment]
@engram-design didn't think to check discussions before posting the issue, sorry about that. I can confirm that running `composer require verbb/hyper:"dev-craft-4 as 1.1.7"` fixes the issue. Thanks!

**Comment by engram-design** [comment]
Fixed in [1.1.8](https://github.com/verbb/hyper/releases/tag/1.1.8)
